### PR TITLE
Feat: start new front

### PR DESCRIPTION
**Pull request by ga-devfront**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Creating _dev folder to build scss and ts with vite. CI and storybook are updated too.
| Type?             | improvement
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | --
| Sponsor company   | @PrestaShopCorp
| How to test?      | Just follow the readme for module and/or storybook to test if all it's fine.

### CODE REVIEW

File: views/templates/layouts/welcome.html.twig
Comment (Quetzacoalt91) [inline views/templates/layouts/welcome.html.twig:16]: Should be translatable

File: views/templates/layouts/welcome.html.twig
Comment (Quetzacoalt91) [inline views/templates/layouts/welcome.html.twig:23]: Should be translatable

File: .github/workflows/build-release.yml
Comment (ga-devfront) [reply .github/workflows/build-release.yml:26]: switch to '20.x'

File: .github/workflows/js.yml
Comment (ga-devfront) [reply .github/workflows/js.yml:17]: switch to '20.x'

File: views/templates/modals/modal-error-report.html.twig
Comment (ga-devfront) [reply views/templates/modals/modal-error-report.html.twig:22]: We going to add translation then we integrate the new pages.

File: views/templates/modals/modal-error-report.html.twig
Comment (ga-devfront) [reply views/templates/modals/modal-error-report.html.twig:21]: same

File: views/templates/modals/modal-error-report.html.twig
Comment (ga-devfront) [reply views/templates/modals/modal-error-report.html.twig:11]: same

File: views/templates/layouts/welcome.html.twig
Comment (ga-devfront) [reply views/templates/layouts/welcome.html.twig:16]: same

File: views/templates/layouts/welcome.html.twig
Comment (ga-devfront) [reply views/templates/layouts/welcome.html.twig:23]: same

**Review by M0rgan01 — APPROVED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

File: controllers/admin/AdminSelfUpgradeController.php
Comment (Hlavtox) [inline controllers/admin/AdminSelfUpgradeController.php:541]: Not sure if this should be in the module files guys, isn't there some other way to run the dev environment?

Comment (ga-devfront) [reply controllers/admin/AdminSelfUpgradeController.php:541]: Unfortunately the Vite dev mode (_dev folder) has no influence on the symfony part, that's why I decided to add an env file to define it manually. But this has no impact on production.

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by AureRita — APPROVED** [review]
Hi @ga-devfront 

Thank you for your PR, I tested it and it seems to works as you can see : 

[recording(286).webm](https://github.com/user-attachments/assets/4e814b01-619a-471c-a582-3ce904c17323)

Tested from : 
1.7.8.9 to 8.1.7 
8.0.4 to 8.1.7 
8.1.5 to 8.1.7 
8.1.7 to 9.0.0
8.1.5 to 9.0.0
8.0.4 to 9.0.0

Because the PR seems to works as expected, It's QA :heavy_check_mark: 

Thank you

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
